android-lib/src/androidTest/java/com/kinvey/androidTest/store/datastore/DataStoreMultiInsertItemsOrderTestKotlin.kt

```diff
@@ -12,11 +12,9 @@ import org.junit.runner.RunWith
 import org.junit.Assert.assertNotNull
 import org.junit.Assert.assertNull
 import org.junit.Assert.assertTrue
-import org.junit.Ignore
 
 @RunWith(AndroidJUnit4::class)
 @SmallTest
-@Ignore("Ignored tests in Kotlin for now while Kotlin migration is not done")
```

Comment:
@b-stolyarov let's investigate why those tests are running ok on the CI server

Reply:
Fixed issues with Kinvey API version that caused some issues in tests.